operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -280,6 +280,7 @@ def __init__(self, env):
   def validate(self):
     self.validateAdminServer()
     self.validateClusters()
+    self.validateServerCustomChannelName()
```

Comment:
I may be missing something, but I don't see a check to make sure that:
-- All defaultChannel in the same configured cluster are either enabled xor all are disabled
-- All SSL Channel in the same configured cluster are either enabled xor all are disabled, and, if enabled, have the same port
-- All admin port in the same configure cluster are either enabled xore all are disabled, and, if enabled, have the same port
-- All nap with same name in configured cluster have same protocol